Src/BootCamp.Chapter/Inventory.cs

```diff
@@ -29,7 +32,53 @@ public void AddItem(Item item)
         /// </summary>
         public void RemoveItem(Item item)
         {
+            if (IsNullOrEmpty(_items) || !IsItemInInventory(item.GetName())) return;
+
+            var updatedItems = new Item[_items.Length - 1];
+            var wasItemFound = false;
+            for (var i = 0; i < _items.Length; i++)
+            {
+                if (!wasItemFound && _items[i].GetName().Equals(item.GetName()))
```

Comment:
Avoid creating trains. A train is this:
```train.Wagon.AnotherWagon.EndOfWagon.Passengers```
cleaner is:
```
train.Wagon
.AnotherWagon
.EndOfWagon
.Passengers
```

Reply:
Right now it might make little sense, but in the future were lines like this tend to drag on, it's useful to keep it consistent.